packages/twenty-front/src/modules/object-record/cache/utils/getRecordsFromRecordConnection.ts

```diff
@@ -5,9 +5,11 @@ import { ObjectRecord } from '@/object-record/types/ObjectRecord';
 export const getRecordsFromRecordConnection = <T extends ObjectRecord>({
   recordConnection,
 }: {
-  recordConnection: RecordGqlConnection;
+  recordConnection: RecordGqlConnection | undefined;
```

Comment:
I'm not sure I like this change. An undefined connection converted to an empty array, I feel we are losing a lot of information